controllers/FundingController.php

```diff
@@ -110,12 +110,25 @@ public function actionEdit()
 
         // Step 1: Wanted Asset Selection
         if ($model->isFirstStep()) {
+
+            // Get Coinsence Community Coin asset to be used as default selected asset
+            $cccSpace = Space::findOne(['name' => 'Coinsence Community Coin']);
```

Comment:
considering that this is a general humhub module, I was wondering if we can use something else as an identifier to select the asset that should be preselected. 
Maybe having something like a default asset? or having this as a global configuration?

Reply:
Hello Michael,

Within the new changes, the default asset is based on global parameter that should be introduced in the humhub parameters file.

You can take a look at the changes.

Reply:
That's great Safouane! I was thinking that it could be done in a module settings page.. but don't do that, it will be just with one configuration, when the time comes when we need a settings page for this module, then we should put this setting there.

Reply:
Karim and the marketing team I think needs this to be deployed tomorrow, that's why i think that we should merge this changes for now and we could do the improvements later.